apps/social/src/pages/channels/Channels.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import { useChannels } from 'hooks';
+import { Loader, Filter } from 'components';
+import { FILTERS } from 'consts';
+import { useAccount } from '@gear-js/react-hooks';
+import { Item } from './item';
+import styles from './Channels.module.scss';
+
+function Channels() {
+  const [filter, setFilter] = useState('All Channels');
+  const { account } = useAccount();
+
+  const { channels } = useChannels();
+  const ownerChannels = channels?.filter((ch) => ch.ownerId === account?.decodedAddress);
+
+  const getList = () => {
+    switch (filter) {
+      case 'My':
+        return ownerChannels;
+      default:
```

Comment:
```suggestion
  const getList = () => {
    switch (filter) {
      case 'My':
        return channels?.filter((ch) => ch.ownerId === account?.decodedAddress);
      default:
```

no need to filter channels without filter